### ISSUE TITLE
Re-order room based on activity

### ISSUE THREAD

**Issue by PureTryOut** [mujx/nheko#2]
As Telegram does, and Riot to an extend, it would be nice to have the room list re-order itself based on activity. So if a room gets a new message, it gets send to the top of the list.

**Comment by jansol** [comment]
But please not instantly - I hate it when I'm trying to go to a room and it jumps away from under my cursor.

It might be a good idea if the room is first marked as active and a small line appears at the top of the list to indicate incoming changes, and only after a short delay is the room actually moved.

**Comment by ara4n** [comment]
the fix i landed on riot-web a few weeks ago is simply to freeze the animation if the mouse is actively moving over the list (with a 100ms timeout or so)

**Comment by kannes** [comment]
Please make this configurable if it gets added, I much prefer a deterministic order of rooms.

**Comment by PureTryOut** [comment]
Seeing there is a setting screen now, I'd still like this implemented. With the amount of rooms I'm in, I have to scroll *a lot* to find some active rooms...
